ui/Dockerfile

```
@@ -1,3 +1,9 @@
+FROM golang:1.23 AS specgen
+WORKDIR /specgen
+COPY ../k8s/migration/swagger ./swagger
```

Right now this docker file is not dynamically generating any openapi.yaml file, so basically we want this file to be generated while we are building this docker file. so that any changes made should be reflected in the UI. 

```suggestion
WORKDIR /specgen
COPY ../k8s/migration/swagger ./swagger
COPY ../k8s/migration/config/crd ./config/crd  

RUN chmod +x ./swagger/generate-openapi.sh && ./swagger/generate-openapi.sh

```